src/custom/utils/trade.ts

```diff
@@ -121,6 +120,8 @@ export async function signAndPostOrder(params: PostOrderParams): Promise<AddUnse
     owner: account,
   })
 
+  const creationTime = new Date().toISOString()
```

Comment:
moved this after the signing is done to reduce the difference with the `creationDate` field from the API if the user takes a few seconds to sign.